packages/react-components/src/components/utils/Datetime.ts

```diff
@@ -114,3 +114,18 @@ const dayToDayName = (day: number, dateStrings: MessageThreadStrings): string =>
       throw new Error(`Invalid day [${day}] passed`);
   }
 };
+
+/**
+ * @private
+ */
+export const generateDefaultTimestamp = (
```

Comment:
nit: I'd keep this helper function in `ChatMessageComponent` related files. This is not generally applicable to anything beyond a message (The `showDate` prop etc are very specific), so no value in moving this to a generic utils file. Instead, moving it here just increases the cross-references in the code-base.

General rule: Keep the modules as self contained as possible. Move generic functionality out to common modules / other modules when it is truly generic (has at least 2 (potential) use-cases in the codebase)